src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -199,6 +204,11 @@ public partial class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.Reso
             ValueFromPipelineByPropertyName = true)]
         public bool? BurstingEnabled { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        public bool? SupportsHibernation { get; set; }
```

Comment:
I'm afraid that the naming of SupportsHibernation is not allowed. Let us discuss the design of cmdlet and parameter in design doc.

Reply:
Design doc is linked. Please let me know if it's still a problem?